server.js

```diff
@@ -7503,6 +7504,13 @@ cache({
           }
         };
         break;
+      case 'yaml':
+        requestOptions = {
+          headers: {
+            Accept: 'application/yaml, text/yaml, text/plain'
```

Comment:
From [this Quora answer](https://www.quora.com/What-is-the-correct-MIME-type-for-YAML-documents/answer/Charles-Banas) it seems we probably should use `'application/x-yaml, text/x-yaml, text/plain'` here. Unless someone has a better source saying something else?

Reply:
github.com, search in code:
- `"text/x-yaml" content type` 39,389 code results
- `"text/yaml" content type` 79,485 code results 
- `"application/x-yaml" content type` 34,264 code results 
- `"application/yaml" content type` 21,948 code results 

Should we add all of them since it's not standardized?

Reply:
Yes that's probably best.